test/test_framework.py

```diff
@@ -1810,7 +1810,7 @@ def test_breakpoint_good_names(self, request: pytest.FixtureRequest, name: str):
             '-',
             '...foo',
             'foo.bar',
-            'bar--' 'FOO',
+            'bar--FOO',
```

Comment:
I wonder if this was a bug, perhaps `'bar--', 'FOO'` was meant?

Reply:
I think so, looking at the [original commit](https://github.com/canonical/operator/commit/11a1849205d750e28aaa4a13938b5864659f928b). Nice find!